mock/src/block.rs

```diff
@@ -280,3 +339,158 @@ impl MockBlock {
         self.to_owned()
     }
 }
+
+#[cfg(test)]
+mod block_tests {
+    use crate::MockBlock;
+    use eth_types::{Address, Bytes, Hash, Word, H64};
+    use ethers_core::types::Bloom;
+    use std::str::FromStr;
+
+    #[test]
+    fn test_blockhash() {
+        // Checking legacy block
+        // curl -X POST --data '{"id":1,"jsonrpc":2.0,"method":"eth_getBlockByNumber","params":["0x61a80",false]}' https://mainnet.infura.io/v3/<API_KEY>
+        let mut mock_block = MockBlock::default();
+        let mock_block = mock_block.parent_hash(
```

Comment:
All the `MockBlock` setter methods take `&mut self`, so there's no need to rebind.  This works as well:
```rust
        mock_block.parent_hash(
            Hash::from_str("0x1e77d8f1267348b516ebc4f4da1e2aa59f85f0cbd853949500ffac8bfc38ba14")
                .unwrap(),
        );
```